config.toml

```diff
@@ -184,3 +184,153 @@ enableRobotsTXT = true
     url = "https://jakartaone.org/"
     parent = "Resources"
     weight = 18
+    
+[[menu.main]]
+    name = "Friends of Jakarta EE"
+    url = "/resources#friends"
+    parent = "Resources"
+    weight = 19
+	
+[languages]
+  [languages.en]
```

Comment:
We need to add a label for the language, as it's used in the language toggle for our pages. Under languages.en we would add the following:
```
  languageName = "English"
  weight =  1 
```

And under languages.zh, we would add something like the following (used Google translate, so it's probably slightly wrong haha):
```
  languageName = "汉语"
  weight =  2
```

An example of this can be seen in the [JakartaOne config file](https://github.com/jakartaee/jakartaone.org/blob/master/config.toml)